.github/workflows/ci.yml

```diff
@@ -7,8 +7,8 @@ on:
       - '!master'
 
 env:
-  BUILDER_VERSION: v0.6.0
-  BUILDER_SOURCE: releases
+  BUILDER_VERSION: fixes
```

Comment:
Is this final or would this change once fixes gets merged?

Reply:
Once the dependent change is merged, I'll update this to a release.